wire-ios-system/Source/ZMSDispatchGroup.h

```diff
@@ -21,16 +21,16 @@
 
 @interface ZMSDispatchGroup : NSObject
 
-@property (nonatomic, readonly, copy) NSString* label;
+@property (nonatomic, readonly, copy, nonnull) NSString* label;
```

Comment:
Maybe `NS_ASSUME_NONNULL_BEGIN`/`NS_ASSUME_NONNULL_END` would make the interface declaration more readable.